frontend/src/data/constants.ts

```diff
@@ -1,4 +1,5 @@
-import { GroupType } from '../types/api-types'
+import { GroupType, OfferStatus } from '../types/api-types'
+import { enumValues } from '../utils/types'
```

Comment:
thought (non-blocking): out of scope for this PR but i'd like to pull the frontend utils folder into the root of the repo for sharing with the backend.

Reply:
I'd like that too, but it's not as straight-forward as it should be because `react-script` doesn't like importing code from outside the `/frontend` directory. I'm sure it's possible to hack around that though.